libraries/SITL/SIM_Gripper_Servo.cpp

```diff
@@ -120,11 +120,7 @@ bool Gripper_Servo::should_report()
         return false;
     }
 
-    if (!is_equal(reported_position, position)) {
```

Comment:
This is an example of where the pattern is broken,

Before you've got a bunch of clauses and a simple default return.  Now you've got a bunch of clauses and a complex default return.

Reply:
Done.